From: Jose Luis Rivero <jrivero@osrfoundation.org>
Date: Tue, 14 Apr 2020 16:31:14 +0200
Subject: do not try to install arial and DejaVuSans fonts.

Forwarded: not-needed

The fonts were removed in the dfsg process and has been replaced using .links
file. Remove cmake instruction about it.
---
 media/fonts/CMakeLists.txt     | 3 +--
 media/gui/fonts/CMakeLists.txt | 1 -
 2 files changed, 1 insertion(+), 3 deletions(-)

diff --git a/media/fonts/CMakeLists.txt b/media/fonts/CMakeLists.txt
index f974026..88e00ed 100644
--- a/media/fonts/CMakeLists.txt
+++ b/media/fonts/CMakeLists.txt
@@ -1,5 +1,4 @@
-SET (files arial.ttf
-           console.ttf
+SET (files console.ttf
            font_arial.png
            font_matisse_itc.png
            Gazebo.fontdef
diff --git a/media/gui/fonts/CMakeLists.txt b/media/gui/fonts/CMakeLists.txt
index 9cfeec8..7b64ce4 100644
--- a/media/gui/fonts/CMakeLists.txt
+++ b/media/gui/fonts/CMakeLists.txt
@@ -1,6 +1,5 @@
 set (files Font.xsd
            DejaVuSans-10.font
-           DejaVuSans.ttf
            FairChar-30.font
            fkp-16.font
            fkp.de.pcf
